source/MRMesh/MRToolPath.h

```diff
@@ -113,6 +112,16 @@ struct ToolPathResult
     Mesh modifiedMesh;
     // constains type of movement and its feed
     std::vector<GCommand> commands;
+    // boundaries of the selection on the original mesh
+    Contours3f contoursBeforeProjection;
+    // boundaries of the working area on the offset mesh (which are a projection of contoursBeforeProjection )
+    Contours3f contoursBeforeCutMesh;
+    // polyline containing start vertices for isolines
+    Contours3f startContours;
+    // polylines were failed to build isolines
+    Contours3f failedContours;
+    // start vertices on the offset mesh used for calcutating isolines
+    std::vector<Vector3f> startVertices;
```

Comment:
mb make debug output optional? (empty pointer in input parameters structure, if set by user then filled)